src/chi/components/mobile-navigation/mobile-navigation.scss

```diff
@@ -0,0 +1,332 @@
+@import '_mixins';
+@import '_variables';
+
+.chi-mobile-nav {
+  .chi-drawer {
+    .chi-mobile-nav__first-level-content {
+      left: 0;
+      position: absolute;
+      transition-duration: 0.3s;
+
+      &.-second-level-shown {
+        left: -23rem;
+      }
+    }
+
+    &.-left,
+    &.-right {
+      width: 23rem;
+    }
+
+    .chi-divider {
+      &.-colorbar {
+        height: 0.25rem;
+      }
+    }
+
+    .chi-mobile-nav__header {
+      align-items: center;
+      background-color: rgba(set-color(grey, 100), 0.5);
```

Comment:
@jllr Are you still planning on moving the theme related styles into an -inverse theme class?

Reply:
@mattnickles yes